app/services/nomis/elite2/api.rb

```diff
@@ -40,6 +41,21 @@ def get_offender_list(prison, page = 0)
 
         ApiPaginatedResponse.new(page_meta, offenders)
       end
+
+      def get_offender(offender_no)
+        route = "/elite2api/api/prisoners/#{offender_no}"
+        response = @e2_client.get(route) { |resp|
```

Comment:
The block here receives two args, 'data' and 'response' ... I think this is working because it's seeing the first one (the json blob) but it might be worth renaming resp to data to be clear?